### PR TITLE
Fix prettier issues

### PR DESCRIPTION
**Pull request by Bruception**
### Description

- Fixes prettier issues
- Setups job to automatically fix style issues and create a PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "build-fe": "cd ./frontend && npm run build-live",
     "pretty": "prettier --check './backend/**/*.{ts,json,js,css,html}' './frontend/**/*.{ts,js,scss}' './frontend/static/**/*.{json,html}'",
     "pretty-code": "prettier --check './backend/**/*.{ts,js,json,css,html}' './frontend/**/*.{ts,js}' './frontend/src/**/*.scss'",
-    "pretty-fix": "prettier --write './backend/**/*.{ts,json,js,css,html}' './frontend/**/*.{ts,js,scss}' './frontend/static/**/*.{json,html}'",
```
Comment (Ferotiq) [inline package.json:21]: Does prettier do scss too? If so I think you should add it there

Comment (Bruception) [reply package.json:21]: we dont have any scss in backend

Comment (Ferotiq) [reply package.json:21]: oh makes sense

**Review by Ferotiq — APPROVED** [review]
LGTM

**Pull request merged**